openmetadata-ui/src/main/resources/ui/src/components/Auth/AuthProviders/AuthProvider.tsx

```diff
@@ -435,9 +435,9 @@ export const AuthProvider = ({
         // Parse and update the query parameter
         const queryParams = Qs.parse(config.url.split('?')[1]);
         // adding quotes for exact matching
-        const domainStatement = `(domain.fullyQualifiedName:${escapeESReservedCharacters(
+        const domainStatement = `(domain.fullyQualifiedName:"${escapeESReservedCharacters(
           activeDomain
-        )})`;
+        )}")`;
```

Comment:
isn't it will create issue with domain with name? cc @karanh37

Reply:
No this will not.